### ISSUE TITLE
GUI stalls at pulling image kalabox/pantheon-edge - nothing works.

### ISSUE THREAD

**Issue by ibuildit** [kalabox/kalabox#1525]
Nothing works so far after trying the recommended install for OSX. This is as far as I get.

![screen shot 2016-08-03 at 20 37 55](https://cloud.githubusercontent.com/assets/5072873/17377497/4271ee66-59ba-11e6-9293-d81de8bb6cb7.png)

**Comment by pirog** [comment]
@ibuildit going to need more helpful feedback to point you in the right direction. 

Have you looked at our docs at all? There are good troubleshooting articles that can help you give us a better idea about what is happening.

**Comment by Mchampsee** [comment]
The operation might have simply timed out. I would delete the directory created and then try running again.

**Comment by break9** [comment]
i tried the latest 13.0 -beta 4

I installed over the previous version tried to pull a site. nothing happened.

so.......

1. I deleted the app the .kalabox ,the docker machines and emptied the trash

2. I restarted the mac 

3. Installed kalabox

4. Created a new machine key for kalabox on pantheon and used it to pull up my pantheon sites. 

5. I tried to pull a site and it hung

6 I tried to pull an additional site it also hung

![screen shot 2016-08-15 at 5 57 26 pm](https://cloud.githubusercontent.com/assets/1411503/17684747/be757918-6312-11e6-93cb-5e44d132b88c.png)
